internal/archive/fsops.go

```diff
@@ -141,15 +137,22 @@ func walk(sourcePath string, baseDir string, archive *zip.Writer, ignore map[str
 			header.Name = filepath.ToSlash(getRelativePath(path, baseDir))
 		}
 
-		// compress file
-		header.Method = zip.Deflate
+		if info.IsDir() {
+			header.Name += "/"
+		} else {
+			header.Method = zip.Deflate
+		}
 
 		// add new header and file to archive
 		writer, err := archive.CreateHeader(header)
```

Comment:
Why are we using `CreateHeader` and not `Create`?

Reply:
CreateHeader recieves header, prepared by zip.FileInfoHeader (with additional properties on it,  missing by Create).